sample/mpp-library/src/commonTest/kotlin/ProfileApiTest.kt

```diff
@@ -30,7 +31,7 @@ class ProfileApiTest {
     @BeforeTest
     fun setup() {
         json = Json {
-            ignoreUnknownKeys = true
+            encodeDefaults = false
         }
```

Comment:
just `json = Json.Default` now will be good :)

Reply:
Fixed